torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -645,7 +645,32 @@ def rotate(
         return rotate_image_pil(inpt, angle, interpolation=interpolation, expand=expand, fill=fill, center=center)
 
 
-pad_image_pil = _FP.pad
+def _parse_pad_padding(padding: Union[int, List[int]]) -> List[int]:
```

Comment:
I've merged the check for invalid types as well as wrong lengths here, since this function is also used by `pad_bounding_box` and that currently doesn't have these checks.